website/docs/jellyfish/api/poolpair.md

```diff
@@ -161,6 +161,29 @@ interface PoolShareOptions {
 }
 ```
 
+## poolSwap 
+Creates a poolSwap transaction.
+
+```ts title="client.poolpair.poolSwap()"
+interface poolpair {
+  poolSwap (metadata: PoolSwapMetadata, utxos: PoolSwapUTXO[] = []): Promise<string>
+}
+
+interface PoolSwapMetadata {
+  from: string
+  tokenFrom: string
+  amountFrom: number
+  to: string
+  tokenTo: string
+  maxPrice?: number
+}
+
+interface PoolSwapUTXO {
```

Comment:
Change this interface
(Refer to https://github.com/DeFiCh/jellyfish/pull/413/files#r668418832)